internal/command/project.go

```diff
@@ -26,38 +26,33 @@ func (c *Commands) AddProjectWithID(ctx context.Context, project *domain.Project
 	if projectID == "" {
 		return nil, zerrors.ThrowInvalidArgument(nil, "COMMAND-nDXf5vXoUj", "Errors.IDMissing")
 	}
-
-	existingProject, err := c.getProjectWriteModelByID(ctx, projectID, resourceOwner)
-	if err != nil {
-		return nil, err
-	}
-	if existingProject.State != domain.ProjectStateUnspecified {
-		return nil, zerrors.ThrowInvalidArgument(nil, "COMMAND-opamwu", "Errors.Project.AlreadyExisting")
+	if !project.IsValid() {
+		return nil, zerrors.ThrowInvalidArgument(nil, "PROJECT-IOVCC", "Errors.Project.Invalid")
```

Comment:
I'm not sure why we switch out `Errors.Project.AlreadyExisting` for `"Errors.Project.Invalid` here?

Reply:
Also on [this line](https://github.com/zitadel/zitadel/pull/9317/files#diff-acb463d04c8aeb5527493ab22b976b767b2720c182c8f5967c7e7bcd92fe502bR203) you seem to call `isProjectStateExists()` for similar purpose. Shouldn't that function be used here also?